Lib/pathlib/_local.py

```diff
@@ -824,7 +823,9 @@ def rmdir(self):
         """
         os.rmdir(self)
 
-    _rmtree = shutil.rmtree
+    def _rmtree(self):
+        import shutil
```

Comment:
Let's leave a comment to explain why, so it's not "tidied up" later:

```suggestion
        # Lazy import to improve module import time
        import shutil
```